### PR TITLE
docs: Create Starters section

### PR DESCRIPTION
**Pull request by carolinamenezes**
## What's the purpose of this pull request?
Add Starter library.

## How it works? 
The `starter.js` page renders the new `StarterPage` component, presenting a Starter Library overview. The data about each starter is being fetched from the `static/data/starters.json` file. For each starter, there is a file inside the new `starters` folder. These pages are rendering the `StarterComponentPage` component.

### CODE REVIEW

File: docs/src/pages/sections/StarterComponentPage/StarterComponentPage.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  owner: 'VTEX',
+  description:
+    'Kickoff your store with this boilerplate. This starter ships with the main FastStore configuration files you need to get up and running blazing fast.',
+  features: [
+    'Landing page',
+    'Product page',
+    'Intelligent search',
+    'Filters for categories',
+    'Optimistic shopping cart',
+    'CSS modules abd Tailwind for straight-forward customization',
```
Comment (gvc) [inline docs/src/pages/sections/StarterComponentPage/StarterComponentPage.tsx:16]: I don't believe we're using CSS modules. Also, there's a small typo.

```suggestion
    'CSS modules and Tailwind for straight-forward customization',
```

**Review by gvc — COMMENTED** [review]
Just a tiny question that I have

File: docs/src/pages/sections/StarterComponentPage/StarterComponentPage.tsx (hunk 1 above)
Comment (carolinamenezes) [reply docs/src/pages/sections/StarterComponentPage/StarterComponentPage.tsx:16]:
```suggestion
```

**Review by tlgimenes — APPROVED** [review]

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -136,6 +151,14 @@ module.exports = {
         {
           title: 'More',
           items: [
+            {
+              label: 'Starters',
+              to: '/starters',
+            },
+            {
+              label: 'Frequently asked question',
```
Comment (carolinamenezes) [inline docs/docusaurus.config.js:159]:
```suggestion
              label: 'Frequently asked questions',
```

**Pull request merged**